cfg/rippled-example.cfg

```diff
@@ -892,6 +939,12 @@
 #   [import_db]     Settings for performing a one-time import (optional)
 #   [database_path]   Path to the book-keeping databases.
 #
+#   There are 4 or 5 bookkeeping SQLite databases that the server creates and
+#   maintains. If you omit this configuration setting, it will default to
+#   creating a directory called "db" located in the same place as your
+#   rippled.cfg file. Partial pathnames will be considered relative to
+#   the location of the rippled executable.
```

Comment:
nit: instead of "it will default to creating a directory", just say, "the server creates a directory." Similarly, instead of "will be considered relative" just say "are relative".

Reply:
Changed and reworded to be less passive tone.